.github/workflows/PULL_REQUEST_TEMPLATE.md

```
@@ -0,0 +1,84 @@
+- Test environments (local/dev/staging).
+- Steps to reproduce testing.
+- Manual test cases (e.g., "Verified login flow on Safari/Firefox").
+- Automated tests added/updated (if applicable).
+
+---
+
+## 📸 Screenshots/Recordings
+
+_Visual proof of UI changes (if applicable). Annotate to highlight changes._  
+**Example:**  
+| Before | After |  
```

markdownlint-cli2 (0.17.2)

Tables should be surrounded by blank lines
null

(MD058, blanks-around-tables)